when creating a bulk action job, prefix druid: to each pid if it does not have one yet

peetucket opened the pull request:
fixes #676 

- [x] needs a test

jcoyne commented:
Rather than having this conditional in the model, is it possible to prevent this bad data (pids without namespaces) from getting to the model level in the first place?  I would like to see something like this in a Form object (think Reform, see ApoForm) or baring that, in the controller.

peetucket commented:
OK, moved logic to controller and updated test to reflect its new location (a form object for one method seemed potentially overkill).  Can squash commits if this approach makes more sense.
